extensions/wikia/MercuryApi/handlers/MercuryApiMainPageHandler.class.php

```diff
@@ -4,16 +4,22 @@
 
 class MercuryApiMainPageHandler {
 
-	public static function getMainPageData( MercuryApi $mercuryApiModel ) {
+	// TODO: remove $newFormat param after release release of XW-2590 (XW-2625)
```

Comment:
For future it'll probably be easier to have a separate method that is Copy&Paste of method that should be removed.